docs/content/latest/benchmark/performance-troubleshooting.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Troubleshooting performance issues in YugabyteDB
+headerTitle: Performance Troubleshooting
+linkTitle: Performance Troubleshooting
+description: Common steps to take when investigating performance issues in YugabyteDB
+image: /images/section_icons/architecture/concepts.png
+headcontent: Performance Troubleshooting
+menu:
+  latest:
+    identifier: large-datasets-2-ysql
+    parent: benchmark
+    weight: 7
+showAsideToc: true
+isTocNested: true
+---
+
+Use this page for general guidance and steps you can take when troubleshooting the performance of your Yugabyte cluster.
+
+## Location of various files on a YB cluster
+
+### YB Software/Binaries on the cluster
+
+The software packages are symlinked at /home/yugabyte/{master|tserver}. Be aware that the master & tserver may be at different versions of the software (e.g. during rolling software upgrades).
+
+You should see something like this:
+```
+$ ls -lrt /home/yugabyte/master
+total 4
+lrwxrwxrwx. 1 yugabyte yugabyte   27 Jan 15 19:27 logs -> /mnt/d0/yb-data/master/logs
+lrwxrwxrwx. 1 yugabyte yugabyte   66 Jan 15 19:28 bin -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/bin
+lrwxrwxrwx. 1 yugabyte yugabyte   66 Jan 15 19:28 lib -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/lib
+lrwxrwxrwx. 1 yugabyte yugabyte   72 Jan 15 19:28 linuxbrew -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/linuxbrew
+lrwxrwxrwx. 1 yugabyte yugabyte   85 Jan 15 19:28 linuxbrew-xxxxxxxxxxxx -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/linuxbrew-xxxxxxxxxxxx
+lrwxrwxrwx. 1 yugabyte yugabyte   71 Jan 15 19:28 postgres -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/postgres
+lrwxrwxrwx. 1 yugabyte yugabyte   68 Jan 15 19:28 pylib -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/pylib
+lrwxrwxrwx. 1 yugabyte yugabyte   68 Jan 15 19:28 share -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/share
+lrwxrwxrwx. 1 yugabyte yugabyte   68 Jan 15 19:28 tools -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/tools
+lrwxrwxrwx. 1 yugabyte yugabyte   65 Jan 15 19:28 ui -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/ui
+lrwxrwxrwx. 1 yugabyte yugabyte   84 Jan 15 19:28 version_metadata.json -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/version_metadata.json
+lrwxrwxrwx. 1 yugabyte yugabyte   66 Jan 15 19:28 www -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/www
+-rw-rw-r--. 1 yugabyte yugabyte    0 Jan 15 19:29 master.out
+-rw-rw-r--. 1 yugabyte yugabyte 2200 Jan 15 20:18 master.err
+drwxr-xr-x. 2 yugabyte yugabyte   25 Jan 15 20:18 conf
+```
+
+### Config Files
+You can find config files for the master and tserver at `/home/yugabyte/{master|tserver}/conf/server.conf`.
+
+### Transaction Logs a.k.a WAL (write-ahead logs)
+You can find WAL files at `/mnt/d*/yb-data/{master|tserver}/wals`.
```

Comment:
for wals/rocksdb, maybe we can prefix both with a note about, assume `--fs_data_dirs=/mnt/d0,/mnt/d1`